raster-foundry/raster-foundry
# [WIP][Platforms] Add team service

## Overview

This PR adds team service and temporary code in team controller for testing purposes.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

## Testing Instructions
 * **Run migrations if needed**
 * Turn off feature flag for platform and organization management.
 * Navigate to page `admin/organization//teams` and open console
 * See if team creation, list, get by id, update, and deletion work and are logged correctly
 * There should not be new records in `teams` table in your dev db after the test since records for test purposes should be deleted at the end by the controller

Closes #3245
